# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fdclock/hotfix-libpng.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- fdclock/configure.ac.vanilla	2025-08-13 12:47:41.795344588 +0200
+++ fdclock/configure.ac	2025-08-13 12:47:45.897782031 +0200
@@ -24,7 +24,7 @@
 AC_CHECK_FUNCS([gettimeofday localtime_r])
 
 PKG_CHECK_MODULES(FDCLOCK, cairo)
-PKG_CHECK_MODULES(PNG, libpng12)
+PKG_CHECK_MODULES(PNG, libpng)
 
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
